reporters_db/data/reporters.json

```diff
@@ -18221,12 +18226,7 @@
             "name": "Ridgely's Notebook",
             "notes": "Dates are rough, but wide, estimated from a series called Delaware Cases, which had these dates. See: http://www.worldcat.org/oclc/2882722",
             "variations": {
-                "Ridgely's Notebook I": "Ridgely's Notebook",
-                "Ridgely’s Notebook I": "Ridgely's Notebook",
-                "Ridgely’s Notebook II": "Ridgely's Notebook II",
-                "Ridgely’s Notebook III": "Ridgely's Notebook III",
-                "Ridgely’s Notebook IV": "Ridgely's Notebook IV",
-                "Ridgely’s Notebook V": "Ridgely's Notebook V"
```

Comment:
whats going on here?

Reply:
I changed this to use a regex to avoid having them as variations, you can see it in each edition. https://github.com/freelawproject/reporters-db/pull/113/files/4754ef5b65e8cead851400ddc8c6e789da7ae2df..867dab20eb6d5e78bc16aa6a546d301d8dedc6fd#diff-45f9b4fa9d04419349ec0fd08c75b92af5cdb7d89cd1c0c27194fa152f249e62R18176